.github/workflows/build.yml

```diff
@@ -11,14 +11,16 @@ on:
   pull_request:
     branches: [ master, develop ]
 
-env:
-  JDK_VERSION: 16
 
 jobs:
   analyze_sources:
     name: "Do we need to build the application?"
-    runs-on: ubuntu-latest
-
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        java-version: [ 16 ]
+        distro: [ 'zulu', 'temurin' ]
```

Comment:
I'm not convinced we need to build on more than kne as it simply takes too much time and that was the main reason this action was not using matrix in the first place.

Reply:
do you want me to change it to just use temurin?

Reply:
Here are the changes to be made:
build.yml

- [ ] use one distro 
- [ ] use remove matrix in favor of env.DISTRIBUTION 'temurin'

nightly.yml
- [ ] use one distro 
- [ ] use remove matrix in favor of env.DISTRIBUTION 'temurin'
 
Let me know. I can probably work on this tomorrow.

Reply:
Yes, unless others contributors object, please stick to one distro as we simply do not need more in this project. Thanks.

Reply:
I made the variable name consitent with what your team had before:
JDK_VERSION
JDK_DISTRO

I thought DISTRIBUTION was too generic and a little long. So, I prepending JDK and shortened the word, to let's the reader of the builds(workflows) environment variables pertain to Java.